ROADMAP.md

```diff
@@ -5,15 +5,9 @@ This document is a general overview of all upcoming development for `kompose`. T
 We track each release by using GitHub [milestones](https://github.com/kubernetes-incubator/kompose/milestones).
 
 ## Upcoming releases
+Kompose is released every the last week of the month.
```

Comment:
I think it should be:
`Kompose is released every last week of the month`

Otherwise, LGTM! :) Merge away when done!